src/components/footer/index.njk

```diff
@@ -117,7 +145,35 @@
         text: 'No'
       },
       {
-        text: 'Array of items to display in the list'
+        text: 'Array of items to display in the list in navigation section of the footer.'
+      }
+    ],
+    [
+      {
+        text: 'navigation.items.[].text'
+      },
+      {
+        text: 'string'
+      },
+      {
+        text: 'No'
+      },
+      {
+        text: 'List item text in the navigation section of the footer.'
+      }
+    ],
+    [
+      {
+        text: 'navigation.items.[].href'
```

Comment:
I think this is still inconsistent with other places using 'item.{}.prop'?

Reply:
yep, fixed it now